upstash/manifest.json

```diff
@@ -0,0 +1,60 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "fe1f17e3-11a4-4e44-b819-8781ebcc86f8",
+  "app_id": "upstash",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "uninstallation": "README.md#Uninstallation",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Publish your Upstash Redis metrics to Datadog",
+    "title": "Upstash",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Upstash Database Metrics",
+        "image_url": "images/upstash-dashboard.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Offering::Integration",
+      "Category::Cloud",
+      "Category::AI/ML",
+      "Category::Data Stores",
+      "Submitted Data Type::Metrics"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "auto_install": true,
+      "source_type_name": "Upstash",
+      "configuration": {},
+      "events": {
+        "creates_events": true
```

Comment:
Does this integration send events into Datadog? if not, this should be false

Reply:
I recently updated this branch according to your comments @anawishnoff